### PR TITLE
[spike] DCOS-46183  ${} inside Trans element

### PR DESCRIPTION
**Pull request by GeorgiSTodorov**
Update translation.

Closes https://jira.mesosphere.com/browse/DCOS-46183

This just updates the message catalog. For testing instructions, screenshots, etc, please see https://github.com/mesosphere/dcos-ui-plugins-private/pull/904 .

### CODE REVIEW

File: locale/zh/messages.json [hunk 1]
```diff
@@ -279,7 +279,7 @@
   "Disk": "磁盘",
   "Disk (MiB)": "磁盘 (MiB)",
   "Disk Allocation": "磁盘分配",
-  "Disk Space (MiB)": "磁盘空间 (MiB)",
+  "Disk Space (Mib)": "",
```
Comment (jaxesn) [inline locale/zh/messages.json]: Not really sure the right way to handle this, but this feels like a weird change to lose the translation for?

Comment (GeorgiSTodorov) [reply locale/zh/messages.json]: Not sure why this was changed.

**Comment by GeorgiSTodorov** [comment]
I had to rebase. @natmegs , @TattdCodeMonkey , please re-review.

**Comment by GeorgiSTodorov** [comment]
I had to rebase again.

**Comment by GeorgiSTodorov** [comment]
Hold merge until the other PR is merged.

**Pull request merged**